packages/manager/modules/vps/src/dashboard/modal/snapshot-download/snapshot-download.controller.js

```diff
@@ -0,0 +1,112 @@
+import 'moment';
+import { CURL_URL, URL_GENERATION_STATUS } from './snapshot-download.constants';
+
+export default class SnapshotDownloadController {
+  /* @ngInject */
+  constructor($translate, atInternet, VpsService) {
+    this.$translate = $translate;
+    this.atInternet = atInternet;
+    this.VpsService = VpsService;
+  }
+
+  $onInit() {
+    this.size = null;
+    this.creationDate = moment(this.tabSummary.snapshot.creationDate).format(
+      'LLL',
+    );
+    this.linkExpirationDate = this.getLinkExpirationDate();
+    this.downloadLink = null;
+    this.downloadLinkCurlCommand = null;
+    this.isLoading = false;
+    this.notificationMessage = null;
+    this.errorMessage = null;
```

Comment:
**nitpick:** Remove unnecessary declarations
```suggestion
    this.isLoading = false;
```

Reply:
It would, in fact, reduce the number of code lines. However, I prefer to keep these declarations this way to have an exhaustive list of the the attributes instantiated in this controller.
Having the controller's attributes declared this way is also a pattern that I see often in the manager.